fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -2008,13 +2027,16 @@ ACTOR Future<Void> updatePersistentData(Reference<TLogGroupData> self,
 		self->persistentData->set(
 		    KeyValueRef(persistRecoveryLocationKey, BinaryWriter::toValue(locationIter->value.first, Unversioned())));
 	}
-
+	// key : persistCurrentVersionKeys + interface.id
+	// value : persistentDataVersion
+	// for groups served by the same interface(implying they have the same generation), they should have the same key.
 	self->persistentData->set(
 	    KeyValueRef(BinaryWriter::toValue(logData->logId, Unversioned()).withPrefix(persistCurrentVersionKeys.begin),
 	                BinaryWriter::toValue(newPersistentDataVersion, Unversioned())));
 	self->persistentData->set(KeyValueRef(
 	    BinaryWriter::toValue(logData->logId, Unversioned()).withPrefix(persistKnownCommittedVersionKeys.begin),
 	    BinaryWriter::toValue(logData->knownCommittedVersion, Unversioned())));
+	// TODO: update teams -> tags mapping when it changes!
```

Comment:
team -> tags mapping never change and is fixed when DD first creates a team. Each team uses a unique ID.

Reply:
yeah, forgot to remove this comment, will do in a separate PR.